### ISSUE TITLE
Getting Started

### ISSUE THREAD

**Issue by howudodat** [h-mdm/hmdm-server#5]
Just getting started, added a phone here are the questions issues I have found:

1. Why do I have to enter the device number, wouldn't this be better as an autoincrement number?
2. it assumes US numbers.  The number I am trying to enter is in the format of: +000 11 222 33 44, where 000 is the country code and the rest the phone number
3. After entering all the information generate QR code is never enabled

Peter

**Comment by h-mdm** [comment]
1. No, it wouldn't. Each company has its own device numbering scheme, it can be an identifier in their ERP, IMEI, serial number, etc.
2. The phone number is different from the device number. Entering the phone number is not required, the system will determine the phone number of the installed SIM card automatically. The phone number format is international without spaces (+xxx112223344 where xxx is the country code).
3. Please read https://qa.h-mdm.com/1298/ to check the reason why QR code is disabled.

**Comment by howudodat** [comment]
1.  Understood
2.  the phone number actually is locked to a one digit country code it seems. the phone number is 10 digits 1122334455 + cuntry code 243 and the last two digits are not enterable  ==> +2 (431) 122-33-44
3.  ok, the qr code is now enabled, however it is tied to server:8080 however my server sits behind a web proxy that forwards from port 80 to 8080, is there a way to configure the produces url or do I have to just manually change it each time?
4.  This created a new situation:  scanning the QR code just shows me some JSON but doesn't do anything with it.  And in looking at the JSON it points to h-mdm.com and to our server:8080 which wont work

**Comment by h-mdm** [comment]
2. You're right, that seems to be an issue. However since you're compiling the app from source code, you can update the phone number UI mask at line 55 of server/src/main/webapp/app/components/main/view/modal/device.html.
3. The URL is configured in the XML configuration file (/var/lib/tomcat8/conf/Catalina/localhost/ROOT.xml or .../hmdm.xml), parameter "base.url". See https://qa.h-mdm.com/1533/ for details on the forwarding of port 80 to 8080.
4. Please read about QR code based enrollment here: https://h-mdm.com/quick-start/ section "Enrolling the device". 

I kindly ask you not to combine multiple questions in one issue in the future. Otherwise, it becomes hard to follow and other people would have difficulties while looking for a solution of similar issues. Please open a new issue for each new issue.

Please also use our Q&A website if you have more questions: https://qa.h-mdm.com
